esf_main.tex

```diff
@@ -891,7 +891,7 @@ \subsubsection{Charging}
 	
 	The accumulator will only be charged on the charging cart. The cart will be able to support the full weight of the accumulator and only move when a dead man's switch is activated.
 	
-	The accumulator cart will contain a specialized shutdown and monitoring circuit that allows the accumulator to be charged through the main pack connector.The shutdown and monitoring circuit will contain a clearly labeled emergency stop push button with a minimum diameter of 25mm which opens the AIRS ands stops charging. It will also communicate with the AMS over CAN and display critical information during charging.
+	The accumulator cart will contain a specialized charging circuit that allows the accumulator to be charged through the main pack connector.The charging circuit will contain a clearly labeled emergency stop push button with a minimum diameter of 25mm which opens the AIRS ands stops charging. It will also communicate with the AMS over CAN and display critical information during charging.
```

Comment:
Charging circuit on the cart that mimics the shutdown circuit in the vehicle? Could be a little clearer.